subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -1125,6 +1140,39 @@ int bt_mesh_cfg_app_key_add(uint16_t net_idx, uint16_t addr, uint16_t key_net_id
 	return cli_wait();
 }
 
+int bt_mesh_cfg_node_reset(uint16_t net_idx, uint16_t addr, uint8_t *status)
+{
+	BT_MESH_MODEL_BUF_DEFINE(msg, OP_NODE_RESET, 2);
+	struct bt_mesh_msg_ctx ctx = {
+		.net_idx = net_idx,
+		.app_idx = BT_MESH_KEY_DEV_REMOTE,
+		.addr = addr,
+		.send_ttl = BT_MESH_TTL_DEFAULT,
+	};
+	int err;
+
+	err = cli_prepare(NULL, OP_NODE_RESET_STATUS);
```

Comment:
Should get the status from the response status, here and in reset status callback.